runtime/doc/editing.txt

```diff
@@ -1651,3 +1651,21 @@ There are three different types of searching:
    with depth limiter (/usr/**2) or upward search (;) notations.
 
  vim:tw=78:ts=8:noet:ft=help:norl:
+
+==============================================================================
+11. Secure Files						*secure*
+
+Secure Files are used to manage files that in a trust database, allowing
+trusted files to be sourced via |exrc|.
+
+							*:trust* *E5570*
+:trust [++deny] [++remove] [{file}]
+			Manage file in the trust database. Without any options,
+			`:trust` will mark file associated with current buffer
+			as trusted.
+			`++deny` will mark a file as denied and you won't be
+			asked if you wan't to source that file.
+			`++remove` will remove the file from the trust database.
+			`++deny` and `++remove` accept optional parameter
+			`{file}`, that is path to the file to be denied or
+			removed.
```

Comment:
Writing this in GitHub's review UI, so you will need to re-wrap it to fit within 79 cols.

```suggestion
			Manage files in the trust database. Without any options or arguments,
			`:trust` adds the file associated with the current buffer
			to the trust database, along with the SHA256 hash of its contents.
			`++deny` marks the file associated with the current buffer (or {file}, if given) as denied; no prompts will be displayed to the user and the file will never be executed.
			`++remove` removes the file associated with the current buffer (or {file}, if given) from the trust database. Future attempts to read the file in a secure setting (i.e. with 'exrc' or |vim.secure.read()|) will prompt the user if the file is trusted.
```

Reply:
Done in d3d5654ca469c73ed196ac902a9e1eaadf4d0ec1